travis-ci/make_rpms.sh

```diff
@@ -1,29 +1,34 @@
 #!/bin/bash
 
 set -e
+
 usage()
 {
     echo "Usage: $0 [<options>]"
     echo
     echo "Options:"
     echo "  -h                              This message"
     echo "  -C                              Skip %check when building RPMs"
+    echo "  -s                              Stop after building SRPMs"
```

Comment:
If I was using this script for the first time, this help message would leave me with a few questions: What exactly am I skipping when I stop after building SRPMs? Are there any relevant tasks that happen before building SRPMs? You could either modify this specific sentence or add a description of the tool to the top of the help message to answer these questions.

Reply:
I will change it to "Build SRPMs only", which is more obvious.